### ISSUE TITLE
DCN - RMSE vs BCELoss

### ISSUE THREAD

**Issue by holy-way** [tensorflow/recommenders#756]
In the DCN tutorial code ([GitHub link](https://github.com/tensorflow/recommenders/blob/main/docs/examples/dcn.ipynb)), the loss function used is Root Mean Squared Error (RMSE). However, I believe thatBinary Cross-Entropy should be used instead.

In the original DCN paper, it specifically mentions that Binary Cross-Entropy was used as the loss function.

Could you explain why there is this difference between the tutorial and the paper?
